eslint 8 throwing error in CI

> Error: Error while loading rule 'import/no-unused-modules': Failed to load plugin 'import' declared in '.eslintrc': Cannot find module '/home/runner/work/eslint-plugin-import/eslint-plugin-import/node_modules/eslint-plugin-import/lib/index.js'. Please verify that the package.json has a valid "main" entry

> Occurred while linting /home/runner/work/eslint-plugin-import/eslint-plugin-import/tests/files/just-json-files/invalid.json
[71](https://github.com/import-js/eslint-plugin-import/actions/runs/3449533983/jobs/5757534657#step:5:72)

See https://github.com/import-js/eslint-plugin-import/actions/runs/3449533983/jobs/5757534657#step:5:66

Happened within the last 4 days.  Last versioned release of eslint was 8 days ago.  Node also release 8 days ago.

--- a/.npmrc
+++ b/.npmrc
@@ -1,1 +1,2 @@
 package-lock=false
+install-links=false

--- a/.travis.yml
+++ b/.travis.yml
@@ -25,6 +25,7 @@ matrix:
   fast_finish: true
 
 before_install:
+  - 'curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.2/install.sh | bash && . $NVM_DIR/nvm.sh'
   - 'nvm install-latest-npm'
   - 'NPM_CONFIG_LEGACY_PEER_DEPS=true npm install'
   - 'npm run copy-metafiles'

